src/lib/i18next/words.ts

```diff
@@ -0,0 +1,261 @@
+  "unused_tariff_plan": "Unused [Tariff Plan]",
+  "used_tariff_plan": "Used [Tariff Plan]",
+  "edit_the_tariff_plan_a_metered_tariff_plan_cannot_be_turned_into_a_notmetered_plan_and_viceversa": "Edit the tariff plan. A metered tariff plan cannot be turned into a not-metered plan, and vice-versa.",
+  "admin_link": "Admin [Link]",
+  "access_user": "Access [User]",
+  "inactive_user": "Inactive [User]",
+  "give_access_to_user": "Give access [to user]",
+  "process_the_users_request_for_access": "Process the user's request for access",
+  "the_decision_you_make_here_is_final_if_you_accept_you_can_revoke_the_users_access_later_by_making_them_inactive": "The decision you make here is final. If you accept, you can revoke the user's access later by making them \"inactive\".",
+  "a_user_with_admin_rights_can_edit_site_settings_tariff_plans_and_users_and_approve_expenses": "A user with admin rights can edit site settings (tariff plans and users) and approve expenses.",
+  "admin_rights_app_user": "Admin rights [App User]",
+  "an_inactive_user_can_not_make_any_changes_in_the_app_nor_access_any_information": "An inactive user can not make any changes in the app nor access any information.",
+  "active_user": "Active [User]",
+  "period": "Period",
+  "payments_site": "Payments [Site]",
+  "reports": "Reports",
+  "paid_by_user": "Paid by [User]",
+  "account": "Account",
+  "bank": "Bank",
+  "new_app_users": "New app users",
+  "payment_amount_site": "Payment Amount [Site]",
+  "payslip": "Payslip",
+  "recipient": "Recipient",
+  "spent_money": "Spent [Money]",
+  "jobs_to_do": "Jobs to Do",
+  "live_issues": "Live Issues",
+  "mee_panyar_profit": "Mee Panyar Profit",
+  "mee_panyar_reimbursement": "Mee Panyar Reimbursement",
+  "meesayar_profit": "Meesayar Profit",
+  "next_deadline": "Next deadline",
+  "rejected_inventory_purchase": "Rejected [Inventory Purchase]",
+  "to_collect_customers": "To Collect [Customers]",
+  "to_meter_customers": "To Meter [Customers]",
+  "total_owed_to_mpy": "Total Owed to MPY",
+  "profit": "Profit",
+  "issues_solved_incidents": "Issues Solved [Incidents]",
+  "issues_unsolved_incidents": "Issues Unsolved [Incidents]",
+  "jobs_done": "Jobs Done",
+  "jobs_not_done": "Jobs Not Done",
+  "mpys_share_profit": "MPY's share [Profit]",
+  "outstanding_payments": "Outstanding [Payments]",
+  "payment_rate_": "Payment Rate (%)",
+  "submitted_automatically_on_date": "Submitted automatically on [date]",
+  "to_approve_inventory_purchase": "To Approve [Inventory Purchase]",
+  "approved_inventory_purchases": "Approved [Inventory Purchases]",
+  "billed_money": "Billed [Money]",
+  "collected_money": "Collected [Money]",
+  "filing_report": "Filing report...",
+  "the_deadline____of_the_current_period__has_passed_a_report_will_be_filed_and_the_site_will_move_to_the_next_reporting_period": "The deadline ([ ])  of the current period ([ ])has passed. A report will be filed and the site will move to the next reporting period."
```

Comment:
Small question: This is something that's expect to be fixed in the Adalo DB right rather than with with the translation script right?

Reply:
Yeah, your understanding is right--words.ts is updated based on what's in the Adalo DB which we treat as our source of truth.